src/api/user/index.ts

```diff
@@ -1,12 +1,12 @@
 import userApi from './userApiClient';
 import {
-  ILoginParams, ILoginResponse, IModifyParams, IRegisterParams, IUser,
+  LoginParams, LoginResponse, ModifyParams, RegisterParams, User,
 } from './entity';
 
-export const register = (param: IRegisterParams) => userApi.post<IUser>('/', param);
+export const register = (param: RegisterParams) => userApi.post<User>('/', param);
 
-export const login = (param: ILoginParams) => userApi.post<ILoginResponse>('/login', param);
+export const login = (param: LoginParams) => userApi.post<LoginResponse>('/login', param);
```

Comment:
보다 보니깐 login 반환값이 없을 때에 대한 핸들링을 사용하는 쪽에서 많이 해주는것 같아요
사용할때마다 관련한 핸들링을 처리하기 번거로우니, login 함수 내부에서 관리할 수도 있을것 같아요
이렇게 되면 사용하는 쪽에선 온전히 data 가 있을 때와 관련한 로직만 작성할 수 있습니다

```suggestion
export const login = async (param: LoginParams) => {
  const { data } = await userApi.post<LoginResponse>("/login", param);

  if (data == null) {
    throw new Error("로그인 데이터가 존재하지 않습니다.");
  }

  return data;
};
```

Reply:
오 좋은 의견 감사합니다. 기존에는 이 파일이 api의 엔드포인트와 타입만 모아두는 느낌으로 생각했는데, 이러한 핸들링은 가독성에 도움을 주겠네요!